Add missing free context in at the end of aes_crypt_xts_size() (#4676)
ARMmbed/mbedtls: ChangeLog.d/issue4176.txt

Bugfix
   * Fix an issue where resource is never freed when running one particular

>      test suite with an alternative AES implementation. Fixes #4176

Reviewer: Missing full stop.

>    * Fix an issue where resource is never freed when running one particular
>      test suite with an alternative AES implementation. Fixes #4176.

Reviewer 2: We're rather picky on good English in changelog entries.

* “Fix an issue where X happens” → ”Fix X“ — the extra words are just a distraction.
* “resource” → “a resource”
* “where resource is never freed” has a name: it's a resource leak
* “when running one particular test suite” → “in a test suite”

```suggestion
   * Fix a resource leak in a test suite with an alternative AES
     implementation. Fixes #4176.
```
